cmd/populator/cmd/register/ocf_manifests.go

```diff
@@ -89,30 +87,83 @@ func runDBPopulate(ctx context.Context, src string) (err error) {
 			err = multierror.Append(err, cErr)
 		}
 	}()
-
 	session := driver.NewSession(neo4j.SessionConfig{})
 	defer func() {
 		if sErr := session.Close(); sErr != nil {
 			err = multierror.Append(err, sErr)
 		}
 	}()
 
-	var gitHash []byte
-	if cfg.UpdateOnGitCommit {
-		log.Info("APP_UPDATE_ON_GIT_COMMIT set. Updating manifests only if git commit changed.")
-		gitHash, err = getGitHash(rootDir)
+	dataInDB, err := dbpopulator.IsDataInDB(session, log, commits)
+	if err != nil {
+		return errors.Wrap(err, "while verifying commits in db")
+	}
+	if dataInDB {
+		return nil
+	}
+	for _, src := range sourcesInfo {
+		err = runDBPopulate(ctx, cfg, session, log, src)
+		if rErr := os.RemoveAll(src.dir); rErr != nil {
+			err = multierror.Append(err, rErr)
+		}
+		if err != nil {
+			return errors.Wrap(err, "while populating db")
+		}
+	}
+	err = dbpopulator.SaveCommitsMetadata(session, commits)
+	if err != nil {
+		return errors.Wrap(err, "while saving metadata into db")
+	}
+
+	return nil
+}
+
+func getSourcesInfo(ctx context.Context, cfg dbpopulator.Config, log *zap.Logger, sources []string) ([]sourceInfo, error) {
+	var sourcesInfo []sourceInfo
+	for _, source := range sources {
+		parent, err := ioutil.TempDir("/tmp", "*-hub-parent")
 		if err != nil {
-			return errors.Wrap(err, "while getting `git rev-parse HEAD`")
+			return nil, errors.Wrap(err, "while creating temporary directory")
 		}
-	} else {
-		log.Info("APP_UPDATE_ON_GIT_COMMIT not set. Ignoring git commit, always updating manifests.")
+
+		dstDir := path.Join(parent, "hub")
```

Comment:
as mentioned above, if not hard-coded to `hub` the parent dir can be one, with multiple folders downloaded from different sources